src/borg/cache.py

```diff
@@ -521,6 +521,10 @@ def __init__(
         self.cache_mode = cache_mode
         self.timestamp = None
         self.txn_active = False
+        self.do_cache = os.environ.get("BORG_USE_CHUNKS_ARCHIVE", "yes").lower() in ["yes", "1", "true"]
+
+        if not self.do_cache:
+            logger.info("Cache for chunks archive is disabled")
```

Comment:
Maybe we better leave that away.
The admin should know well if they set that env var.

Output can sometimes completely disturb some functionality, e.g. if the normal output is some piece of json and suddenly there is that log message in front of it.

Reply:
I will remove it, it is not really necessary.

Does borg sends json to stdout and logs to stderr ? That way logs should never interfere with data.

Reply:
I've seen test failing due to such stuff. Guess at least some test capture stderr and stdout together into 1 string.